docs/abi.md

```diff
@@ -81,27 +81,28 @@ functions** as
 [WebAssembly imports](https://webassembly.github.io/spec/core/syntax/modules.html#imports)
 (all of them defined in the `oak` module):
 
-- `wait_on_channels: (usize, u32) -> u32`: Blocks until data is available for
-  reading from one of the specified channel handles. The channel handles are
-  encoded in a buffer that holds N contiguous 9-byte chunks, each of which is
-  made up of an 8-byte channel handle value (little-endian u64) followed by a
-  single channel status byte. Invalid handles will have an `INVALID_CHANNEL`
-  status, but `wait_on_channels` return value will only fail for internal errors
-  or if _all_ channels are invalid.
+- <a name="wait_on_channels">`wait_on_channels: (usize, u32) -> u32`</a>: Blocks
```

Comment:
Maybe we should just use headers for these, e.g.

```md
### `wait_on_channels`

Signature: `(usize, u32) -> u32`

Blocks until ...
```

Up to you.

Reply:
Yeah, that's a better idea &ndash; done.